documentation/asciidoc/accessories/ai-camera/model-conversion.adoc

```diff
@@ -0,0 +1,87 @@
+== Model Deployment
+
+The process of deploying a new neural network model to the Raspberry Pi AI Camera will normally consist of the following steps:
+
+. A neural network model must be provided.
+. The model must be quantised and compressed so that it can be run using the resources available in the IMX500 camera.
+. The compressed model must be converted to IMX500 format.
+. Finally, the model must be packaged into a firmware file that can be loaded at runtime into the camera.
+
+The first three steps will normally be performed on a more powerful computer such as a desktop or server, whilst the final packaging step must be performed on a Raspberry Pi.
```

Comment:
```suggestion
To deploy a new neural network model to the Raspberry Pi AI Camera, complete the following steps:

. Provide a neural network model.
. Quantise and compress the model so that it can run using the resources available on the IMX500 camera module.
. Convert the compressed model to IMX500 format.
. Package the model into a firmware file that can be loaded at runtime onto the camera.

The first three steps will normally be performed on a more powerful computer such as a desktop or server. The final packaging step must be performed on a Raspberry Pi.
```